atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -1191,6 +1194,22 @@ private void renderGetRowsColumnRange(boolean isDynamic) {
                 } line("}");
                 line("return transformed;");
             } line("}");
+            line();
+            line("@Override");
+            line("public Iterator<Map.Entry<", Row, ", ", ColumnValue, ">> getRowsColumnRange(Iterable<", Row, "> rows, ColumnRangeSelection columnRangeSelection, int batchHint) {"); {
+                line("Iterator<Map.Entry<Cell, byte[]>> results = t.getRowsColumnRange(getTableRef(), Persistables.persistAll(rows), columnRangeSelection, batchHint);");
+                line("return Iterators.transform(results, e -> {"); {
+                    line(Row, " row = ", Row, ".BYTES_HYDRATOR.hydrateFromBytes(e.getKey().getRowName());"); 
+                    if (isDynamic) {
+                        line(Column," col = ", Column, ".BYTES_HYDRATOR.hydrateFromBytes(e.getKey().getColumnName());");
+                        line(table.getColumns().getDynamicColumn().getValue().getJavaObjectTypeName(), " val = ", ColumnValue, ".hydrateValue(e.getValue());");
+                        line(ColumnValue, " colValue = ", ColumnValue, ".of(col, val);");
+                    } else {
+                        line(ColumnValue, " colValue = shortNameToHydrator.get(PtBytes.toString(e.getKey().getColumnName())).hydrateFromBytes(e.getValue());");
+                    }
+                    line("return new AbstractMap.SimpleEntry<", Row, ", ", ColumnValue, ">(row, colValue);");
```

Comment:
Might consider `Maps.immutableEntry` To prevent setValue (and avoid some generics cruft).